test/sandbox/app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
+require "ostruct"
```

Comment:
Since OpenStruct is being extracted I'd love to remove it completely. Thoughts on updating the tests to use actual classes instead of OpenStruct? (maybe PORO's in the test apps `models` directory?

Reply:
I think thats wise